benchmarking/__main__.py

```diff
@@ -188,10 +188,14 @@ def run_pyctest():
     # replace any type of whitespace with single space
     pyctest.BUILD_NAME = " ".join(pyctest.BUILD_NAME.split())
     # how to build the code
-    pyctest.BUILD_COMMAND = "{} setup.py install".format(
-        pyctest.PYTHON_EXECUTABLE)
-    # generate the code coverage
+    pyctest.BUILD_COMMAND = " ".join([
+        pyctest.PYTHON_EXECUTABLE, "-m",
+        "pip", "install", pyctest.SOURCE_DIRECTORY, "-vvv",
+        # "--target", pyctest.BINARY_DIRECTORY,
+    ])
```

Comment:
Jonathan says that pip adds noise to the build logs on CMake.



Try the following.

a. Use `python setup.py build -b BINARY_DIRECTORY` so tomopy is installed to a custom location instead of to the site-packages

b. set `WORKING_DIRECTORY` to `BINARY_DIRECTORY/libs` so the benchmarks are executed in a location that can discover tomopy

Thus no annoying messages for you and no annoying installation mess for me.